src/config/parser.rs

```diff
@@ -35,18 +35,7 @@ pub struct Config {
     pub upstream_search_engines: Vec<crate::engines::engine_models::EngineHandler>,
     pub request_timeout: u8,
     pub threads: u8,
-    pub safe_search: u8,
-}
-
-/// Configuration options for the aggregator.
-///
-/// # Fields
-///
-/// * `random_delay` - It stores the option to whether enable or disable random delays between
-/// requests.
-#[derive(Clone)]
-pub struct AggregatorConfig {
-    pub random_delay: bool,
+    pub rate_limter: RateLimiter,
```

Comment:
```suggestion
    pub rate_limiter: RateLimiter,
```

Reply:
I think this could have been a `good-first-issue` because I think this is just a small change, not much. So I think it would have been better to have it as a `good-first-issue`. But anyway since you have committed this change already, so it is ok, but I think next time if it is a small change then we can have it as a `good-first-issue`. What do you say ??